pyclesperanto_prototype/_tier4/__init__.py

```diff
@@ -1 +1,2 @@
-from ._connected_components_labeling_box import connected_components_labeling_box
\ No newline at end of file
+from ._connected_components_labeling_box import connected_components_labeling_box
+from ._get_sorensen_dice_coefficient import get_sorenson_dice_coefficient
```

Comment:
Is it Sorensen or Sorenson?

Reply:
https://en.wikipedia.org/wiki/S%C3%B8rensen%E2%80%93Dice_coefficient